### DISCUSSION TITLE
commented "main" stop extension

### DISCUSSION THREAD

**Discussion by hugo-levet** [sibiraj-s/vscode-scss-formatter#13] [Q&A]
My code with commented "main"  is stopping

This code works:
```
div {
    background: red;
}
```

This code does not work:
```
// main
div {
    background: red;
}
```

Why it's not work? It's a bug or it's a normal?

**Comment by sibiraj-s** [comment]
Hi, Formatting works fine for me. Do you see any errors in the output?
